From 17c2dbd411ce0b2221b5559c3c0eff01920dea40 Mon Sep 17 00:00:00 2001
From: Max Reitz <mreitz@redhat.com>
Date: Mon, 13 Mar 2017 17:47:19 +0100
Subject: [PATCH 18/24] qemu-iotests: Disable 030, 040, 041

RH-Author: Max Reitz <mreitz@redhat.com>
Message-id: <20170313174719.29543-1-mreitz@redhat.com>
Patchwork-id: 74280
O-Subject: [RHEL-7.4 qemu-kvm PATCH 9/9] qemu-iotests: Disable 030, 040, 041
Bugzilla: 1427176
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>

Upstream: N/A

All of these tests require (and test) live block operations, 030 and 041
also need blkdebug support. Both of these features are disabled
downstream, so the tests need to be disabled, too.

Signed-off-by: Max Reitz <mreitz@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 tests/qemu-iotests/group | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/qemu-iotests/group b/tests/qemu-iotests/group
index 58b3d05..c1fc89d 100644
--- a/tests/qemu-iotests/group
+++ b/tests/qemu-iotests/group
@@ -36,7 +36,7 @@
 027 rw auto quick
 028 rw backing auto
 029 rw auto quick
-030 rw auto backing
+# 030 rw auto backing -- requires blkdebug and block jobs
 031 rw auto quick
 032 rw auto
 033 rw auto quick
@@ -46,8 +46,8 @@
 037 rw auto backing
 038 rw auto backing
 039 rw auto
-040 rw auto
-041 rw auto backing
+# 040 rw auto -- requires block jobs
+# 041 rw auto backing -- requires blkdebug and block jobs
 042 rw auto quick
 043 rw auto backing
 044 rw auto
-- 
1.8.3.1

